device/arista/x86_64-arista_7050_qx32/platform.json

```diff
@@ -183,7 +157,466 @@
             {
                 "name": "qsfp32"
             }
+        ],
+        "thermals": [
+            {
+                "name": "Cpu temp sensor"
+            },
+            {
+                "name": "Board sensor"
+            },
+            {
+                "name": "Front-panel temp sensor"
+            },
+            {
+                "name": "Rear temp sensor"
+            },
+            {
+                "name": "Power supply 1 inlet temp sensor"
+            },
+            {
+                "name": "Power supply 1 internal sensor"
+            },
+            {
+                "name": "Power supply 2 inlet temp sensor"
+            },
+            {
+                "name": "Power supply 2 internal sensor"
+            }
         ]
     },
-    "interfaces": {}
-}
\ No newline at end of file
+    "interfaces": {
```

Comment:
Can you use "version sort" so Ethernet100, Ethernet104 go after the two-digits?

Reply:
Done